### DISCUSSION TITLE
Inquiry Regarding the Selection of Temporal Resolution for Hydrodynamic Data in Parcels

### DISCUSSION THREAD

**Discussion by Caiyuan6** [Parcels-code/Parcels#1544] [Q&A] [answered]
Dear Erik et al.

I'm currently engaged in a research project using Parcels to simulate MP dynamics in aquatic environments. 

I've been diving into Tutorials & Documentation, and relevant literature, including works by Delandmeter and van Sebille (2019) and Lange and Van Sebille (2017). However, I haven't been able to find information on how to appropriately handle temporal resolution for hydrodynamic data in Parcels. Ensuring the hydrodynamic data's temporal resolution aligns with the needs of my microplastics research is paramount.

Could you please provide guidance on where I can find relevant sections in the code or documentation? I'm eager to gain a deeper understanding of how to select suitable temporal resolution for hydrodynamic data in Parcels, especially for my microplastics research.

Thank you in advance for your help!
 
Best regards,
Cyuan

**Comment by erikvansebille** [comment]
Hi @Cyuan-hub, you could try the article by [Qin _et al_ [2014]](https://www.sciencedirect.com/science/article/pii/S1463500314000158?via%3Dihub), which discusses temporal resolution needed for 1/10degree global ocean models.

**Reply by Caiyuan6** [reply]
Hi @erikvansebille,

Thank you for your suggestion! I thoroughly review the article by [Qin _et al_ [2014]](https://www.sciencedirect.com/science/article/pii/S1463500314000158?via%3Dihub) to gain insights into the temporal resolution needed for global ocean models.

Additionally, I am curious to know what interpolation method Parcels uses for the temporal resolution of hydrodynamic data. Does it rely on linear interpolation, or are other methods utilized? I would greatly appreciate clarification on this aspect.

Thank you once again for your assistance.

Best regards,
Cyuan

**Reply by erikvansebille** [reply, marked as answer]
Yes, Parcels uses linear time interpolation between consecutive hydrodynamic snapshots

**Reply by Caiyuan6** [reply]
OK, Thank you for clarifying!
